# Object (0x60A, 0x60B, 0x60C, 0x60D)

uint8 ObjectID
geometry_msgs/PoseWithCovariance ObjectPosn
geometry_msgs/TwistWithCovariance ObjectVrel
geometry_msgs/AccelWithCovariance ObjectArel
int8 ObjectDynProp
float32 ObjectRCS
int8 ObjectMeasState
int8 ObjectProbOfExist
int8 ObjectClass
float32 ObjectOrientationAngle
float32 ObjectLength
float32 ObjectWidth